go/models/deploy_files.go

```diff
@@ -38,6 +38,9 @@ type DeployFiles struct {
 
 	// functions
 	Functions interface{} `json:"functions,omitempty"`
+
+	// functions config
+	FunctionsConfig interface{} `json:"functions_config,omitempty"`
```

Comment:
it feels a bit whacky to put `interface{}` here, because that's a very nondescript type - but it seems like all other fields do the same :shrug: if others feel similar, maybe we can open an issue to make this a little more precise.